hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -56,17 +57,23 @@ if {$FOUNDRY_ROOT != ""} {
 set ri_create_outputs_in_create_env true
 set ri_print_module_nand2_counts true
 # enable analysis of large arrays
-set ri_max_total_range_bits 100000
+set ri_max_total_range_bits 20
+set ri_prefer_liberty true
+
+############################
+## Apply Liberty File ##
+
+read_liberty /workspaces/repo/hw/top_earlgrey/ip/ast/lib/ast.lib -db_dir ./RI_compiled_libs/verix_2019_a_p27_2_2022_03_20
```

Comment:
ah one more hardcoded path here :)

Reply:
for the first part, we can probably do `$CDC_WAIVER_DIR/hw/top_earlgrey/cdc/ast.lib`, given that the LIB file is checked into `hw/top_earlgrey/cdc/ast.lib`.

Reply:
for the second part, is it necessary to add `./RI_compiled_libs/...` to this command?
this makes this quite tool version dependent, and we typically do not expose this in scripts that are checked in...

Reply:
it looks like the `read_liberty` command falls back to a default if the `db_dir` switch is not used. can we just remove this argument?

Reply:
1. I replaced /workspaces/repo/hw/top_earlgreyip/ast/lib/ast.lib with /workspaces/repo/hw/top_earlgrey/ip/ast/lib/ast.lib
2. ./RI_compiled_libs/verix_2019_a_p27_2_2022_03_20 is a temporary storage for saving ast.v. This argument was requested by RI folks

Reply:
once the hardcoding path here is fixed, i think we are good to merge!

Reply:
i hope the db_dir ends up in the scratch area somewhere... hehe.

Reply:
Thanks for adding the AST_LIB, that part LGTM!

For the second part, if this is just scratch storage and the name can be arbitrary, can we name this differently and remove the tool version from the name? If not explicitly needed, it would be good to avoid checking explicit tool version names...

We could name it e.g. ./RI_compiled_libs/verix_libs...